nautobot_golden_config/views.py

```diff
@@ -201,21 +201,24 @@ def queryset_to_csv(self):
         """Export queryset of objects as comma-separated value (CSV)."""
 
         def convert_to_str(val):
-            if bool(val) is False:  # pylint: disable=no-else-return
+            if val is None:
+                return "N/A"
+            if bool(val) is False:
                 return "non-compliant"
-            elif bool(val) is True:
+            if bool(val) is True:
                 return "compliant"
-            return "N/A"
+            return None
 
         csv_data = []
-        headers = sorted(list(models.ConfigCompliance.objects.values_list("rule__feature__name", flat=True).distinct()))
+        # Use set instead of order_by() as Mysql has issues with distinct and order_by.
```

Comment:
distinct should eliminate the reason for a set

Reply:
distinct isn't doing that from testing though.  Was playing with it for awhile yesterday and never could get distinct to work and actually eliminate duplicats unless i used `order_by()`